aeon/dj_pipeline/analysis/block_analysis.py

```diff
@@ -188,37 +183,14 @@ def make(self, key):
         )
         patch_keys, patch_names = patch_query.fetch("KEY", "underground_feeder_name")
```

Comment:
Should we exclude Dummy patches here or is this already handled elsewhere and we can assume dummy patches will never be fetched?

Reply:
Not excluded, Dummy Patches will still be included in the analysis, but will return 0 pellets, wheel data will still be computed